tests/integration/targets/resources/aliases

```diff
@@ -0,0 +1 @@
+resources
```

Comment:
I guess you should also add the following lines to have the test running in the CI.

```
cloud/aws
zuul/pravic
```

Reply:
done